apps/common/controllers/start.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
-var util = require('util');
-var controllers = require('hof').controllers;
-var BaseController = controllers.base;
+const BaseController = require('hof').controllers.base;
 
-var StartController = function StartController() {
-  BaseController.apply(this, arguments);
-};
+module.exports = class StartController extends BaseController {
 
-util.inherits(StartController, BaseController);
+  constructor(options) {
+    super(options);
+  }
 
-StartController.prototype.getValues = function getValues(req) {
-  req.sessionModel.reset();
-  BaseController.prototype.successHandler.apply(this, arguments);
-};
+  getValues(req, res, callback) {
+    req.sessionModel.reset();
+    super.successHandler(req, res, callback);
```

Comment:
It's purely personal preference, but I prefer `super.successHandler.apply(this, arguments);` I also think that there is less opportunity for any of the arguments to be accidentally omitted